corehq/apps/app_manager/static/app_manager/js/modules/module_view.js

```diff
@@ -68,8 +76,10 @@ hqDefine("app_manager/js/modules/module_view", function () {
         };
 
         var showCaseTypeChangedWarning = function () {
-            $('#case_type_changed_warning').css('display', 'block');
-            $('#case_type_form_group').addClass('has-error');
+            if (casesExist) {
+                $('#case_type_changed_warning').css('display', 'block');
```

Comment:
Nitpick, but I'd use `$(...).addClass("hide")` - it's feasible that future changes could restyle this to use a display other than `block` and that'll break this.

Reply:
updated